ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2

```diff
@@ -17,3 +17,24 @@ management api gnmi
    provider eos-native
 {%     endif %}
 {% endif %}
+{% if management_api_gnmi.transport is arista.avd.defined %}
+{%     if management_api_gnmi.transport.grpc is arista.avd.defined %}
+{%         for transport in management_api_gnmi.transport.grpc %}
+{%             if transport.name is arista.avd.defined %}
+   transport grpc {{ transport.name }}
+{%             endif %}
+{%             if transport.vrf is arista.avd.defined %}
+      vrf {{ transport.vrf }}
+{%             endif %}
+{%             if transport.notification_timestamp is arista.avd.defined %}
```

Comment:
EOS output shows `ip access-group` first, and `notification timestamp` after. Please reorder.